docs/tutorials/getting-started-with-csharp/console-teleprompter.md

```diff
@@ -32,7 +32,7 @@ There are a lot of features in this tutorial. Let’s build them one by one.
 You’ll need to setup your machine to run .NET core. You can find the
 installation instructions on the [.NET Core](https://www.microsoft.com/net/core)
 page. You can run this
-application on Windows, Ubuntu Linux, OS X or in a Docker container. 
+application on Windows, Ubuntu Linux, macOS or in a Docker container. 
```

Comment:
should we just say Linux here instead of Ubuntu Linux @blackdwarf? Since we do support other distros